Cargo.toml

```diff
@@ -1,4 +1,5 @@
 [workspace]
+resolver = "2"
```

Comment:
Should we split that and the change for edition 2021 into a separate PR? Or is it relevant to fixing the lints?

Reply:
It fixes another warning, not a "fatal" one, though. I'll move it to a separate PR (and merge it directly).